Core/Src/CAN/general_can.c

```diff
@@ -48,10 +50,14 @@ void CAN_general_receive_callback(FDCAN_HandleTypeDef* hfdcan, uint32_t RxFifo0I
         }
         idgeneral = RxHeader.Identifier;
```

Comment:
```suggestion
        uint32_t id= RxHeader.Identifier;
```

tira a declaração em cima